### PR TITLE
Adding MDM setting to hide custom servers

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

**Comment by bhariharan** [comment]
Verified by hardcoding.

**Comment by kchitalia** [comment]
Looks good to me

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostStorage.m [hunk 1]
```diff
@@ -60,19 +60,23 @@ + (SFSDKLoginHostStorage *)sharedInstance {
...
+
+        // Load from managed preferences (e.g. MDM).
         if (managedPreferences.hasManagedPreferences) {
-            //If ther are any existing login hosts, remove them as MDM should take highest priority and only the hosts enforced by MDM should be in the list.
+
+            /*
+             * If there are any existing login hosts, remove them as MDM should take
+             * highest priority and only the hosts enforced by MDM should be in the list.
+             */
             if([self.loginHostList count] > 0) {
```
Comment (sgoldberg-sfdc) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostStorage.m]: If MDM is used for an configuration, it also has to be used to provide login hosts?

Comment (bhariharan) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostStorage.m]: Yes - that's a requirement. There's another key for that.

**Pull request merged**